tools/tasks/css_task.ts

```diff
@@ -1,9 +1,11 @@
 import { Task } from './task';
+import Config from '../config';
 
 export abstract class CssTask extends Task {
 
+    // Is it still usefull if cache used ?
   shallRun(files: String[]) {
-    return files.some(f =>
+    return Config.ENABLE_SCSS || files.some(f =>
```

Comment:
`ENABLE_SCSS` is not a precondition for execution of this task. We want to run it only if any of the `css`, `scss`, `sass` files have changed.